src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import logo from "../../assets/transparent-logo.png";
+import "../Navbar/Navbar.css";
+import {
+  ProfileSvg,
+  OutlinedHeartSvg,
+  OutlinedCartSvg,
+} from "../Reusable-Svgs/svgs";
+import { Link } from "react-router-dom";
+import { useData } from "../../context/DataProvider";
+import { useAuth } from "../../context/AuthProvider";
+import { SearchBar } from "../../components/SearchBar/SearchBar";
+
+export function Navbar() {
+  const {
+    state: { itemsInCart, itemsInWishlist },
+  } = useData();
+
+  const { isUserLoggedIn, username } = useAuth();
+
+  return (
+    <nav className="navbar-area">
+      <div className="navbar">
+        <Link to="/">
+          <img className="brand-logo" src={logo} alt="brand-logo" />
+        </Link>
+
+        <ul className="nav-links">
+          <Link
+            to="/products"
+            style={{
+              textDecoration: "none",
```

Comment:
Since doing `textDecoration:"none"` and changing color of link is the  behavior we want  everywhere, you can create a class for it and reuse it.

Reply:
yes will do it